ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -117,6 +117,11 @@ export class ReposList extends React.Component<
         method: string;
         currentRepo: models.Repository;
         displayEditPanel: boolean;
+        sortProperty: 'asc' | 'desc';
```

Comment:
Could you check if this property is used? Check also `filteredRepos`

Reply:
I think it was added again after rebase.
It's right that it's not being used!